pipelines/enterprise-contract.yaml

```diff
@@ -50,11 +50,12 @@ spec:
           value: "$(params.SNAPSHOT)"
         - name: SSL_CERT_DIR
           value: "$(params.SSL_CERT_DIR)"
-        # Tekton only creates results for successful tasks. Since the goal of this pipeline
-        # is not to gate an operation, but to provide feedback to users, do not fail if there
-        # are violations. This ensures users can view the full validation report.
+        # It's confusing for users to see a passing taskrun that represents a failing EC test.
+        # For that reason let's have the taskrun fail when there are EC violations. Also, if
+        # this is set to false (IIUC), it's not possible to have the IntegrationTest gate the
```

Comment:
+1 to having a safer default. But I also think the integration folks should be explicit about which strict level they want. My understanding at the time was that the verify-ec task in the integration pipeline should behave like the tests in the build pipeline. They always pass, but produce a report. It is, of course, a different approach than what the release pipeline does. Have you checked with the integration folks?

Reply:
https://issues.redhat.com/browse/STONE-574 talks about the bigger picture, including the new UX that uses the HACBS_TEST_OUTPUT results to show status to the user. So I think this is okay as a short term thing.